src/core/lib/gprpp/per_cpu.h

```diff
@@ -51,23 +57,39 @@ class PerCpuOptions {
   size_t max_shards_ = std::numeric_limits<size_t>::max();
 };
 
+class PerCpuShardingHelper {
+ protected:
+  size_t GetShardingBits() {
```

Comment:
The refresh mechanism could use a brief explanation. Why can't it be a fixed value for the lifetime of each thread using that PerCpu object?